apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -294,6 +294,13 @@ def prepare_tag(
             )
 
         tag, depends = get_graph_info(html_tags)
+        spec_char_list = list("*&^$@!~[]{}()|:@?/\\")
```

Comment:
```suggestion
        SPEC_CHAR_SET = frozenset("*&^$@!~[]{}()|:@?/\\")
```
Since this is a list that you don't intend to modify, you should make this a constant defined at the top of the file and use a `frozenset` instead, as this prevents modification and leads to faster access time.

Reply:
using a frozen set is a good idea, I don't really care for it being at the top of the file

Reply:
other minor detail: I would prefer not to have the type in the variable name, it should just be `SPEC_CHARS`

Reply:
or better yet, `SPECIAL_CHARS`